src/contracts/payloads/PermissionedPayloadsController.sol

```diff
@@ -0,0 +1,88 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IPayloadsControllerCore} from './interfaces/IPayloadsControllerCore.sol';
+import {PayloadsControllerCore} from './PayloadsControllerCore.sol';
+import {PayloadsControllerUtils} from './PayloadsControllerUtils.sol';
+import {WithPayloadsManager} from './WithPayloadsManager.sol';
+import {IPermissionedPayloadsController} from './interfaces/IPermissionedPayloadsController.sol';
+
+contract PermissionedPayloadsController is
+  PayloadsControllerCore,
+  WithPayloadsManager,
+  IPermissionedPayloadsController
+{
+  function initialize(
+    address owner,
+    address guardian,
+    address initialPayloadsManager,
+    UpdateExecutorInput[] calldata executors
+  ) external {
+    initialize(owner, guardian, executors);
+    _updatePayloadsManager(initialPayloadsManager);
+  }
+
+  function EXPIRATION_DELAY()
+    public
+    pure
+    override(PayloadsControllerCore, IPayloadsControllerCore)
+    returns (uint40)
+  {
+    return 10 days;
+  }
+
+  function GRACE_PERIOD()
+    public
+    pure
+    override(PayloadsControllerCore, IPayloadsControllerCore)
+    returns (uint40)
+  {
+    return 5 days;
+  }
+
+  function MIN_EXECUTION_DELAY()
+    public
+    pure
+    override(PayloadsControllerCore, IPayloadsControllerCore)
+    returns (uint40)
+  {
```

Comment:
is there a reason on why its set to 0? this will make it so that the payload gets executed immidiately. Provably makes sense to still mantain 1, so as to give time to cancel the payload

Reply:
As I wrote in the description, I'm not sure about these constants. Anyway, 1 day looks too much. Do you know who can know the answer?

Reply:
@eboadom

Reply:
Wouldn't make sense to have it as a setter and by guardian?

Reply:
Unresolving as this still set at 0, when it should be bigger

Reply:
Had conversation with Ernesto, decided to make it 0